Screen/ListScreen.tsx

```diff
@@ -1,15 +1,10 @@
 import React from 'react';
 
-import {View} from 'react-native';
 import Weather from '../Components/weather';
 
 class ListScreen extends React.Component {
```

Comment:
Is this component really useful, all I see is that ListScreen is the exact same component as Weather